tabpy-server/server_tests/test_config.py

```diff
@@ -21,6 +21,28 @@ def assert_raises_runtime_error(message, fn, args={}):
         assert err.args[0] == message
 
 
+def append_logger_settings_to_config_file(config_file):
```

Comment:
Just as a tip you'd be much better off just using a StringIO or BytesIO instanced than an actual file for the tests. This could get rather wonky